src/commands/addBinding/settingSteps/LocalAppSettingListStep.ts

```diff
@@ -33,10 +32,11 @@ export class LocalAppSettingListStep extends BindingSettingStepBase {
     public async promptCore(context: IBindingWizardContext): Promise<BindingSettingValue> {
         const localSettingsPath: string = path.join(context.projectPath, localSettingsFileName);
         const settings: ILocalSettingsJson = await getLocalSettingsJson(context, localSettingsPath);
-        const existingSettings: string[] = settings.Values ? Object.keys(settings.Values) : [];
+        const existingSettings: [string, string][] = settings.Values ? Object.entries(settings.Values) : [];
+
         let picks: IAzureQuickPickItem<string | undefined>[] = [{ label: localize('newAppSetting', '$(plus) Create new local app setting'), data: undefined }];
-        picks = picks.concat(existingSettings.map((s: string) => { return { data: s, label: s }; }));
-        const placeHolder: string = localize('selectAppSetting', 'Select setting from "{0}"', localSettingsFileName);
+        picks = picks.concat(existingSettings.map((s: [string, string]) => { return { data: s[0], label: s[0], description: s[1] }; }));
```

Comment:
> I added the value to the list of local settings.

The value can be a secret, though. You could add an option or something to show values, but they should be hidden by default just to be safe. Just like how app settings are currently displayed in the tree view

Reply:
![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/f66ff8e6-22ab-4c32-89f0-2debe982ae81)
![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/8d67472a-cc27-43f4-ad0f-46c8fddd97c9)